<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSpellCheckerWindow</class>
 <widget class="QDialog" name="QtSpellCheckerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="spellChecker">
     <property name="text">
      <string>Spell Checker Enabled</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Dictionary Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="pathContent"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pathButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="currentLanguage">
     <property name="text">
      <string>Current Language:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="currentLanguageValue">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QListWidget" name="languageView"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="language">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
